traffic_ops/app/db/patches.sql

```diff
@@ -39,3 +39,25 @@ ALTER TABLE deliveryservice ADD CONSTRAINT routing_name_not_empty CHECK (length(
 -- altering any enum values within a transaction block and by default
 -- goose runs migrations within a transaction.
 ALTER TYPE profile_type ADD VALUE IF NOT EXISTS 'GROVE_PROFILE';
+
+-- not enough to make boolean default false -- enforce that API can't set it to NULL
+UPDATE cachegroup
```

Comment:
all these columns are of type boolean and default=false?

Reply:
depends how the code is written,   and the only case I know of currently is when created using the kabletown script,  which seeds the db thru sql.  but,  as I don't know of a case where this happens during the normal course of operation,  I'm going to close this and the related issue:   we have more important issues to worry about..